### PR TITLE
Highfi: Topic overview

### LINKED ISSUE #742: Highfi: Topic Overview
TopicOverview needs to be pixel perfect with the mocks.
Portrait Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-

Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/4e4d2aec-fd30-4c20-91ed-daf434f47a89/TP-Info-

Mocks have been created for MDPI Devices.
Make sure that the screen passes the accessibility test also.
For reference you can checkout: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by chrk2205**
## Explanation
- Fixes #742 
## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Sarthak2601** [comment]
@chrk2205, your PR title should be `Fix #742 : Highfi Topic Overview`

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (aggarwalpulkit596) [inline .idea/codeStyles/Project.xml:165]: Don't push changes made in the system files

Comment (chrk2205) [reply .idea/codeStyles/Project.xml:165]: @aggarwalpulkit596 
shall i close this PR and create another one??

**Comment by chrk2205** [comment]
@rt4914 
please check my PR.

**Pull request closed without merging**